src/utils/parseFunctionArguments.js

```diff
@@ -56,7 +55,7 @@ export function mapToKeyValue(nodes) {
     return acc;
   }, []);
 
-  return head(keyVal);
+  return keyVal[0];
```

Comment:
Need a confirmation if this can be null or undefined. If so I should add a fallback `[]`